CMR/python/cmr/auth/token.py

```diff
@@ -0,0 +1,317 @@
+"""
```

Comment:
This file is very long and contains some things that could be moved into a separate util or helper file.

Reply:
there are some I can move out and into the util file, some generic file handling functions, that will shorten it a little.

Reply:
removed 5 functions, 70 lines of code to util module